### ISSUE TITLE
FATAL error when generating mgmt-plane web service

### ISSUE THREAD

**Issue by weidongxu-microsoft** [Azure/autorest.modelerfour#159]
**Describe the issue**

FATAL error when generating mgmt-plane web service:
```
FATAL: YAMLException: bad indentation of a sequence entry in "code-model-v4.yaml" at line 40766, column 25:
                            value: auditlogs
                            ^
```

Interactive screenshot 
![interactive](https://user-images.githubusercontent.com/53292327/74024076-0395c080-49dc-11ea-86d1-4c2941f3f00b.jpg)

YAML
[(421)+code-model-v4.yaml.txt](https://github.com/Azure/autorest.modelerfour/files/4170370/421.%2Bcode-model-v4.yaml.txt)

```
                    - type: ServiceIncident
                      endTime: '2017-11-06T22:21:48Z'
                      message: Your application process was restarted as application environment variables changed. This can most likely occur due to update in app settings or swap operation. This event occurred multiple times during the day.
                      metaData:
                      - - name: feature
                        value: auditlogs
                      - name: displayedName
                        value: Check Audit Logs
                      priority: 0
                      solutions: []
```

** REQUIRED ** 
**ModelerFour** version:4.5.176
**AutoRest-core** version:3.0.6207
**CommandLine** :`--java --payload-flattening-threshold=1`
OpenAPI/Swagger file:
[readme.md.temp.txt](https://github.com/Azure/autorest.modelerfour/files/4170357/readme.md.temp.txt)

**Comment by fearthecowboy** [comment]
This is new!

Oh my.  I'll look into it.

**Comment by weidongxu-microsoft** [comment]
Appear to be fixed by latest core + modelerfour.

**Comment by weidongxu-microsoft** [comment]
Sorry, I think I can still reproduce.

The JSON causing the issue is in example (kind of array in array), so I should be able to workaround it by removing these examples in my local.

https://github.com/Azure/azure-rest-api-specs/blob/master/specification/web/resource-manager/Microsoft.Web/stable/2019-08-01/examples/Diagnostics_ExecuteSiteAnalysis.json#L30
```
                  "metaData": [
                    [
                      {
                        "name": "feature",
                        "value": "auditlogs"
                      },
                      {
                        "name": "displayedName",
                        "value": "Check Audit Logs"
                      }
                    ]
                  ],
```

Reproduced multiple times with updated autorest / core / m4

```
AutoRest code generation utility [cli version: 3.0.6175; node: v10.18.1, max-memory: 8192 gb]
Core: 3.0.6220
M4: 4.6.200
Cmd line: autorest-beta --version=3.0.6220 https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/web/resource-manager/readme.md --java --payload-flattening-threshold=1 --tag=package-2019-08
```

**Comment by fearthecowboy** [comment]
Please update to 4.10.250+ and see if it's still there. I can't go back and fix 4.6.x

**Comment by weidongxu-microsoft** [comment]
Tried. I think it is fixed on 4.10.+.

**Comment by weidongxu-microsoft** [comment]
Sorry, but still reproducible on m4 4.13.348

autorest  3.0.6187
aurorest core 3.0.6282

```
FATAL: YAMLException: bad indentation of a sequence entry in "prechecked-openapi-document.yaml" at line 3436, column 25:
                            value: auditlogs
                            ^
  Error: Plugin modelerfour reported failure.
```
